alerts-server/app/com/alexitc/coinalerts/models/UserVerificationToken.scala

```diff
@@ -13,12 +11,3 @@ object UserVerificationToken {
     UserVerificationToken(token)
   }
 }
-
-case class SendVerificationTokenModel(email: UserEmail)
```

Comment:
This was supposed to be used for re-sending email verification tokens but for some reason, I never completed the implementation, thanks.